modules/compiler/builtins/cmake/FindBuiltinsTools.cmake

```diff
@@ -96,6 +96,13 @@ function(find_builtins_tools tools_dir)
       IMPORTED_LOCATION ${BUILTINS_LINKER})
   endif()
 
+  if (CMAKE_CROSSCOMPILING AND OCK_IN_LLVM_TREE)
+    # If we are building OCK in LLVM TREE, ImportLLVM was not included and
+    # so LLVM_PACKAGE_VERSION is not defined.
+    set(LLVM_PACKAGE_VERSION
+    "${LLVM_VERSION_MAJOR}.${LLVM_VERSION_MINOR}.${LLVM_VERSION_PATCH}${LLVM_VERSION_SUFFIX}")
+  endif()
+
```

Comment:
Ah, that shows what it's for... We could instead perhaps just use `${LLVM_VERSION_MAJOR}.${LLVM_VERSION_MINOR}.${LLVM_VERSION_PATCH}` (note: no `LLVM_VERSION_SUFFIX`) below instead of `LLVM_PACKAGE_VERSION`? We can probably drop the `string(REPLACE "svn"` as well.

Reply:
I've done it, thank you